### DISCUSSION TITLE
High-Availability Across Two Servers - Shared Media Library From NAS

### DISCUSSION THREAD

**Discussion by amacleod457** [AzuraCast/AzuraCast#6136] [Q&A]
Hello all,
I am working on a setup for a high-availability Azuracast deployment with two servers, with both using a common shared media library either CIFS or NFS mounted from a NAS.
The idea is to have a complete fallback if the primary should fail.
I've got everything set up, mapped and mounted, and it seems to work ok, accept the media scanning clashes and conflicts with each box, and the .albumart folder is being clobbered and killed and refilled, as the two machines compete to build their media libraries.

My question is, is there a way to store .albumart outside of the parent media folder, and also a way to assign a unique ID to each .albumart , so two installs can build separate folders based on the same media?

Thanks much for your responses.
Adam

**Comment by Moonbase59** [comment]
If it helps, I simply NFS [mount the external music into a _subfolder_ of the media folder](https://github.com/AzuraCast/AzuraCast/issues/6088#issue-1588645084), and leave the station meddling with its dot folders (i.e., _don’t_ mirror those). No conflicts, same media.

**Comment by BusterNeece** [comment]
@amacleod457 We now have a rudimentary high-availability setup in AzuraCast as of stable version 0.19.0. Any number of AzuraCast installations can share a single database, and they will collectively automatically negotiate one server to take care of synchronized tasks, while allowing you to manage things from any of the servers.

**Reply by druff99** [reply]
Is there instructions or documentation how to set this up?
